pvlib/iotools/pvgis.py

```diff
@@ -390,9 +391,32 @@ def read_pvgis_hourly(filename, pvgis_format=None, map_variables=True):
     raise ValueError(err_msg)
 
 
+def _coerce_and_rotate_pvgis(pvgis_data, tz, year):
+    """
+    After converting TZ, rotate indices so timeseries starts at midnight
+    and force all indices to a common year. Only works for integer TZ.
+    """
+    # assert tz//1 == tz
+    tzname = f'Etc/GMT{-tz:+d}'
+    # check if February is leap year
+    feb1 = pvgis_data[pvgis_data.index.month == 2].index.year[0]
+    if calendar.isleap(feb1):
+        # replace Feb year with a non-leap year
+        pvgis_data.index = [
+            ts.replace(year=1990) if ts.month == 2 else ts
+            for ts in pvgis_data.index]
```

Comment:
Can skip all this if we reduce the options.

Reply:
If you always coerce the year to 1990 there will be no exception.

Reply:
i think the order of operations has to be
1. Sanitize February
2. Rotate hours
3. Concert time zone
4. replace years

If I try to coerce years first then when I convert time zone I get two years.

i can try to experiment with the order but this was what seemed to work best. Let me play around with it some more and maybe I can get rid of step 1. Although I do really like the option to coerce to any year not just 1990.

Reply:
I found a way to make it work without sanitizing February and with `np.roll` by replacing the year and time zone at the same time. Any other way always returns two years.